internal/ansible/controller/reconcile_test.go

```diff
@@ -596,3 +564,15 @@ func TestReconcile(t *testing.T) {
 		})
 	}
 }
+
+// Tech Debt: If we continue to use fake client, convert "reconcile" into a typed object for
+// testing to identify the status sub resource.
```

Comment:
Create an issue for this?

Reply:
Captured here: https://github.com/operator-framework/ansible-operator-plugins/issues/39